app/src/main/java/com/framgia/wsm/screen/requestoff/RequestOffViewModel.java

```diff
@@ -1,18 +1,29 @@
 package com.framgia.wsm.screen.requestoff;
 
+import android.content.Context;
+import android.databinding.BaseObservable;
+import android.databinding.Bindable;
```

Comment:
ERROR: Unused import: android.databinding.Bindable. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports